python/enthic/app.py

```diff
@@ -178,23 +172,73 @@ def pre_cast_integer(probe):
     return str(probe) if probe.__class__ is int else probe if probe.isnumeric() is True else None
 
 
-def result_array(probe, limit, offset=0):
+def ape_list(ape_code):
+    """
+    APE code are not save in the original format.
+    This function returns the corresponding code in MySQL or None
+    if the code does not exist
+
+        :param ape_code : the list of APE for he filter
+        :param APE_CODE : all known APE code
+
+        :return: list of corresponding  APE_CODE in MySQL
+    """
+    ape = list()
+    for ap in ape_code.split(","):
+        for i in APE_CODE:
+            if re.match(ap,APE_CODE[i][0]):
```

Comment:
Please once again use autopep8, a space should be after the coma. Please use it!

Reply:
I've installed autopep8 but it doesn't work.

Reply:
GET /company/search/page?ape=4781Zddd should end up in HTTP 404 but is treated like GET /company/search/page
I know  that I have to  raise an error like "Le code ape n'existe pas", but I were waiting fro in order to decide where we have to raise this error

Reply:
It should be abort(NotFoundJSONResponse()) like in https://github.com/phe-sto/enthic/blob/a294518c978a91e6891ca5377ddf88da38f3f36f/python/enthic/database/mysql_data.py

Reply:
Okay

Reply:
Une erreur est maintenant bien levée lorsque l'argument "ape" ne correspond à aucun code APE connu.